Migrations/02_sample_init_createBlogItem.ts

```diff
@@ -22,29 +22,29 @@ const migration: MigrationModule = {
             .upsertLanguageVariant()
             .byItemId(itemResponse.data.id)
             .byLanguageCodename('default')
-            .withElements([
-                {
+            .withData((builder) => [
+                builder.textElement({
```

Comment:
@Enngage - is there any preference for building elements - with/without a builder?